Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Intrinsics.X86;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Hello first person, what's your first name?");
```

Comment:
Extract the body of for loop in a function. And 
A: just call that function twice (loop becomes not needed for this specific case)
B: call the function in the loop (adds clarity)